lang/lang-add.sh

```diff
@@ -7,7 +7,12 @@
 #  lang_add.txt
 # Updated files:
 #  lang_en.txt and all lang_en_xx.txt
-#
+
+# List of supported languages
+LANGUAGES="cz de es fr it pl"
+
+# Community languages
+LANGUAGES+=" nl" #Dutch
```

Comment:
I have changed that a bit see https://github.com/prusa3d/Prusa-Firmware/blame/MK3_3.11.0/lang/lang-import.sh#L45-L59